### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by godybuoy**
Things done;
1. Add desktop header.
2. Add desktop footer.
3. configured linters.

### CODE REVIEW

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @godybuoy  

## Good job so far!

## There are some issues that you still need to work on to go to the next project but you are almost there!

cat-driving-serious

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Your designs look very good! 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by godybuoy** [comment]
Things done;
1.Update about and skills section.
2.Update contact footer background image.
3.Fix form section.
4.Fix skill section.
4.Configured linters.

File: styles.css [hunk 1]
```diff
@@ -301,3 +305,274 @@ hr {
...
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .work_content img {
+    float: left;
+    width: 48%;
+  }
+
```
Comment (BregornOriginal) [inline styles.css:469]: - [ ] Kindly check your about and skills sections, when I'm up to 1500px wide, the skills sections are going to hide back to the form section, try removing your top property in your contact class selector 🥇 

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @godybuoy  

## Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✅ Linters are passing
✅ Good commit history

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by godybuoy** [comment]
Things done;
1. Fix about and skills section to make it responsive.
2. Configured linters.

File: styles.css [hunk 2]
```diff
@@ -301,3 +305,270 @@ hr {
...
+  #pic2,
+  #pic4 {
+    flex-direction: row-reverse;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
```
Comment (franclobo) [inline styles.css:479]: - **[ OPTIONAL ]** Kindly add the margins around the section and the border radius at the right top to be similar to the Figma design 👇🏽 

| Figma design | Your design |
|---|---|
| image | image |

**Review by franclobo — APPROVED** [review]
Hi @godybuoy, 

# STATUS: APPROVED 😃 

gif-good-job-79

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**